R/cox.zph.mids.R

```diff
@@ -0,0 +1,107 @@
+# Method to test for violation of proportional odds for a cox-ph model on multiple imputations using cox.zph
```

Comment:
In the current implementation I would refrain from calling this function a method.

If you want to make it a method, my suggestion would be to change the implementation in a way that the user just calls cox.zph() _[not cox.zph.mids()]_ on an object of the class "fit.mult.impute" and "coxph" and the function automatically calls the correct generic, either the usual cox.zph() or cox.zph.mids(). However, afaik this would require us to implement our own version of cox.zph().

I've changed the description of the function to class this function a "function" instead of a "method" for now.